config/config-matterwick.default.json

```diff
@@ -29,5 +29,22 @@
     "FileJSON": true,
     "FileFormat": "",
     "FileLocation": ""
+  },
```

Comment:
also I dont see the aws keys to setup

Reply:
Hmm. I now realize I don't use those anywhere, I think the aws library is using the environment variables in my local container? I'll have to see how to pass these explicitly